cloud/pkg/common/client/client.go

```diff
@@ -32,10 +32,11 @@ import (
 )
 
 var (
-	initOnce      sync.Once
-	kubeClient    kubernetes.Interface
-	crdClient     crdClientset.Interface
-	dynamicClient dynamic.Interface
+	initOnce       sync.Once
+	kubeClient     kubernetes.Interface
+	crdClient      crdClientset.Interface
+	dynamicClient  dynamic.Interface
+	authKubeConfig *rest.Config
```

Comment:
please add comment for this `authKubeConfig`, The name can not tell what the variable does